### PR TITLE
blade-371

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: cli/build.gradle [hunk 1]
```diff
@@ -36,7 +35,7 @@ artifacts {
 }
 
 check {
-	dependsOn testExecutableJar, testVersionCommand
+	dependsOn testVersionCommand
```
[the remark this answers is not in the document]
Comment (lawrence-lee) [reply cli/build.gradle]: @gamerson Where is the original `testVersionCommand`

File: cli/blade-jar-tests.gradle [hunk 2]
```diff
@@ -0,0 +1,107 @@
+apply plugin: "java"
```
[the remark this answers is not in the document]
Comment (lawrence-lee) [reply cli/blade-jar-tests.gradle]: So i just add `apply from: rootProject.file("blade-jar-tests.gradle")` to `cli/build.gradle`?

[the remark this answers is not in the document]
Comment (gamerson) [reply cli/blade-jar-tests.gradle]: should be able to just do `apply from: "blade-smoke-tests.gradle"` . does that work?

File: cli/build.gradle (hunk 1 above)
[the remark this answers is not in the document]
Comment (gamerson) [reply cli/build.gradle]: Looks like nowhere Haha, so we can just remove the check { depends on } and the task testVersionCommand above.

**Pull request closed without merging**